src/pgduckdb_guc.cpp

```diff
@@ -119,7 +119,7 @@ char *duckdb_postgres_role = strdup("");
 
 int duckdb_maximum_threads = -1;
 char *duckdb_maximum_memory = strdup("4GB");
-char *duckdb_disabled_filesystems = strdup("LocalFileSystem");
```

Comment:
Why don't we want that by default anymore?

Reply:
Moved this change to a follow up PR